shared/chat/conversation/input-area/normal/platform-input.native.js

```diff
@@ -324,7 +324,8 @@ const styles = Styles.styleSheetCreate({
   },
   input: {
     flex: 1,
-    fontSize: metaData['BodyBig'].fontSize,
+    // Override BodyBig's default weight.
+    fontWeight: Styles.globalStyles.fontRegular.fontWeight,
```

Comment:
We need the line height that `PlainInput` takes from `textType` (changed above). So, we'll override the weight instead of the height.